app/sidekiq/lighthouse/create_intent_to_file_job.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'lighthouse/benefits_claims/service'
+require 'lighthouse_intent_to_file/tag_sentry'
+require 'lighthouse_intent_to_file/monitor'
+
+module Lighthouse
+  class CreateIntentToFileJob
+    include Sidekiq::Job
+    include SentryLogging
+
+    class CreateIntentToFileError < StandardError; end
+
+    # Only pension form configured to create async ITFs for now
+    ITF_FORMS = {
+      # '21-526EZ' => 'compensation',
+      # '21P-530' => 'survivor',
+      # '21P-530V2' => 'survivor',
+      '21P-527EZ' => 'pension'
+    }.freeze
+
+    # retry for one day
+    # exhausted attempts will be logged in intent_to_file_queue_exhaustions table
+    sidekiq_options retry: 14, queue: 'low'
+    sidekiq_retries_exhausted do |msg, error|
+      form_type, form_start_date, veteran_icn = msg['args']
+      itf_log_monitor = LighthouseIntentToFile::Monitor.new
+      itf_type = ITF_FORMS[form_type]
+      user_account = UserAccount.find_by(icn: veteran_icn)
+
+      itf_log_monitor.track_create_itf_exhaustion(itf_type, form_start_date, user_account&.id, error)
+
+      # create ITF queue exhaustion entry
+      itfqe = IntentToFileQueueExhaustion.create({
+                                                   form_type:,
+                                                   form_start_date:,
+                                                   veteran_icn:
+                                                 })
+      ::Rails.logger.info(
+        "IntentToFileQueueExhaustion id: #{itfqe.id} entry created", {
+          intent_to_file_queue_exhaustion: itfqe
+        }
+      )
+    end
+
+    ##
+    # Create an Intent to File using the Lighthouse ITF endpoint for given form type and ICN
+    #
+    # On success do nothing for now
+    # Raises CreateIntentToFileError
+    #
+    # @param [String] form_type
+    # @param [ActiveSupport::TimeWithZone] form_start_date
+    # @param [String] veteran's ICN
+    #
+    def perform(form_type, form_start_date, veteran_icn)
+      init(form_type, form_start_date, veteran_icn)
+
+      @itf_log_monitor.track_create_itf_begun(@itf_type, form_start_date, @user_account.id)
+      @service.create_intent_to_file(ITF_FORMS[form_type], '')
```

Comment:
if ya get here, you could just use `@itf_type` correct? and could include a `@ssn` (which will be set in init)

Reply:
yes to ur first question, will change that. And in order to reliably set the ssn, we would need to make an additional API call to the MPI endpoint, since we can't reliably use the `@current_user` object, especially during retry attempts. I think we decided to omit passing in the SSN for now, and if we need to fetch SSN later on, we would do that as part of our "manual processing" script.